core/src/main/java/org/renjin/util/CDefines.java

```diff
@@ -404,7 +405,7 @@ public static String _(String s) {
   }
   
   public static SEXP findVar(SEXP devName, SEXP env) {
-    return ((Environment)env).findVariable((Symbol)devName);
+    return ((Environment)env).findVariable(Native.currentContext(), (Symbol)devName);

```

Comment:
This method is _only_ called from `GECurrentDevice`, which is in turn called by the `@Internal` `par()` function, so there will be no native context set, and this method will always through an exception.

This class was originally added as an aide to translating C code from GNU R by hand, and has probably outlived its usefulness. Better to have `GECurrentDevice` call `findVariable` directly and just remove this method altogether, along with findVarInFrame below which is not used at all.